external/wiki/Contributing.md

```diff
@@ -31,7 +31,7 @@ Here is a detailed description of the [commit workflow](Workflow) (including [re
 
 ### Contact us!
 
-You can contact us on [Matrix or the mailing list](https://github.com/cockpit-project/cockpit/wiki/About) or by discussing issues in their respective GitHub thread. If an issue has a developer assigned, you may wish to check with that developer to see if they are already working on the issue.
+You can contact us on [irc or the mailing list](https://github.com/cockpit-project/cockpit/wiki/About) or by discussing issues in their respective GitHub thread. If an issue has a developer assigned, you may wish to check with that developer to see if they are already working on the issue.
```

Comment:
Whoops -- no. I think @allisonkarlitskaya patched the /external pages, and I didn't spot that. I did that change on https://github.com/cockpit-project/cockpit/wiki/Contributing instead, and will re-run the workflow.